source/main.cpp

```diff
@@ -252,18 +277,59 @@ Result DownloadTitle(std::string titleId, std::string encTitleKey, std::string t
 
     printf("Starting - %s\n", titleName.c_str());
 
+    // If in install mode, download/install the SEED entry
+    if (selected_mode == install_direct)
+    {
+        // Download and install the SEEDDB entry if install mode
+        // Code based on code from FBI: https://github.com/Steveice10/FBI/blob/master/source/core/util.c#L254
+        // Copyright (C) 2015 Steveice10
+        u8 seed[16];
+        static const char* regionStrings[] = {"JP", "US", "GB", "GB", "HK", "KR", "TW"};
+        u8 region = CFG_REGION_USA;
+        CFGU_GetSystemLanguage(&region);
```

Comment:
You can also access the kagiya server directly from your PC:
https://kagiya-ctr.cdn.nintendo.net/title/0x000400000016DE00/ext_key?country=EU

You'll see that both EU and US return the same crypto key: http://imgur.com/lLlyeP2
I'm not sure why Nintendo requires the country code, honestly.